### ISSUE TITLE
Password prompter broken on amd64 windows build?

### ISSUE THREAD

**Issue by dlenski** [github/hub#963]
I'm using the [precompiled amd64 binary for v2.2.1](https://github.com/github/hub/releases/download/v2.2.1/hub-windows-amd64-2.2.1.tar.gz) on Win7.

When hub prompts me for a a new username/password combination, it **doesn't actually wait for me to finish typing**, but seems to limit the password entry to 4 or 5 characters. Possibly this is a Windows terminal control bug?

```bash
$ hub clone myproject
github.com username: dlenski
github.com password for dlenski (never stored): [hidden]
```

I've tested this with both enterprise and public Github, using the `cmd.exe` shell, Git Bash, and MingW bash... all broken.

I grepped through the source code and found the necessary YAML format of `~/.config/hub`, which I was then able to create manually. **It might be worth explaining how to construct this file manually in the docs**, especially since a lot of `hub` CLI users may already have an OAuth token manually generated.

```yaml
github.com:
- user: USERNAME
  oauth_token: GITHUB_OAUTH_TOKEN
  protocol: https
```

**Comment by mislav** [comment]
This is supposed to already be fixed with #925 and we'll try to release it soon. In the meantime you can try compiling hub from source.

**Comment by dlenski** [comment]
Thanks, looks like #836 is another dup bug report... strange that everyone is reporting different character limits. Mine seems to be the shortest :-P
